### ISSUE TITLE
Show scroll indicator

### ISSUE THREAD

**Issue by Vladimir-csp** [johanmalm/jgmenu#34]
When menu is scrollable, show indicator using symbols: `▼▼▼` or `▲▲▲` near the edge that leads to more items.

**Comment by johanmalm** [comment]
I did something similar in commit [452e0e](https://github.com/johanmalm/jgmenu/commit/452e0e5f810a94a457b25038bf11c2fdee9aa07f)
It's quite unobtrusive as it sits in the 'menu padding' area.

Quite happy to do a text string, but it would have to use up some space in the 'item area', so would take some re-writing.

**Comment by Vladimir-csp** [comment]
Menu padding can be set to 0, so it is not a good place for such an indicator.

BTW, it seems that menu padding has wrong priority, values from tint2 can not be overridden in jgmenu config.

**Comment by johanmalm** [comment]
Sure. I rather liked the 'minimalist' feel and simple code, but I take your point. The item area would be better. Maybe we should add a config option such as scroll_indicator_{top,bottom} which would understand some key words like "line", but would also be used to specify something like "▼▼▼"

Regarding the tint2rc priority, global config variables are set in the following order (i.e. bottom
of list has higher precedence):

    default (config.c)
    jgmenurc (or as specified by --config-file=)
    command line arguments
    tint2rc

Values obtained from tint2rc can be overridden by setting "tint2_rules = 0".

(From JGMENU-CONFIG(1))

Maybe tint2_rules is a bit non-intuitive? My reason for adding it was to make jgmenu align with tint2 even if the user had an existing jgmenurc file.
I guess we could remove it and just comment out ('#') all the lines in docs/jgmenurc and make `jgmenu_run init` add '#' at the beginning of each line too (it already does it additions).

**Comment by Vladimir-csp** [comment]
yes, it would make sense if overrides would go in this order (from low to high): builtin defaults (config.c), automatic guesses (tint2),  config (accounting for #27), command line arguments.

**Comment by kensully** [comment]
Is it possible to add auto scroll while mouse is on the last item in the list?

**Comment by johanmalm** [comment]
Thanks for your feedback. I like that idea. I think that could be done with a timer without much difficulty. And I think it could work well with the original post of this issue.
I'm having a period of ironing out minor snags and cleaning the code base, so might not do it for a little while.
